app/services/notifications/setup/submit-ad-hoc-licence.service.js

```diff
@@ -45,6 +46,34 @@ async function go(sessionId, payload) {
   return {}
 }
 
+/**
+ * Ad-hoc licence notifications has highlighted an issue with using the returns period dates.
+ *
+ * This code is intended as a temporary 'fix' to allow ad-hoc returns to use the returns reminders Notify template
+ * (which requires the 'determinedReturnsPeriod' dates)
+ *
+ * This function alters the 'dueDate' but adding 28 days (this may not be the final state and should be regarded as a
+ * placeholder).
+ *
+ * This also, manually, sets gets the 'startDate' and 'endDate' for 'allYear' which represents the current financial
+ * kyear.
```

Comment:
Apologies, but I don't understand what this comment refers to (plus typos!)

Reply:
The current financial year is calculated by the 'allYear'. So the start and end are used from that returns period. 

No we know theses are going i can temporarily set them to the same as due date and remove this part ?

Reply:
Made a suggested change to the function, which means I think we can drop this comment altogether.